specification/entities.yaml

```diff
@@ -72,6 +72,19 @@ components:
       required:
         - data_source_oddrn
 
+    RelationshipList:
+      type: object
+      properties:
+        data_source_oddrn:
+          type: string
+          example: //aws/glue/123456789010/
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/Relationship'
+      required:
+        - items
```

Comment:
Also here I think `data_source_oddrn` must be required, but not items. We can get the result with 0 relationships, we still will perform a request as I understand, so we have to know where exactly.

Reply:
done